src/components/LoginRegister.vue

```diff
@@ -0,0 +1,195 @@
+<template>
+  <section
+    class="bg-gray-50 dark:bg-gray-900 flex min-h-full min-w-full items-center justify-center py-12 px-4 sm:px-6 lg:px-8 flex-row"
```

Comment:
Is it dark mode version? 🤓

Reply:
It is, if we´re able to have both modes then we can use the dark option, if not, we can just remove one of the modes